pyproject.toml

```diff
@@ -29,7 +29,7 @@ dependencies = [
     "pika >= 1.2.0",
     "dataset",
     "pymongo > 3.0",
-    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@3.1.0.dev1",
+    "sdx-pce @ git+https://github.com/atlanticwave-sdx/pce@fix/sdx-controller-443",
```

Comment:
this is temporary while we wait for PR https://github.com/atlanticwave-sdx/pce/pull/274 goes on the review/approval process.